package.json

```diff
@@ -51,11 +51,11 @@
     "rollup-plugin-node-resolve": "^4.2.4",
     "rollup-plugin-replace": "^2.2.0",
     "rollup-plugin-terser": "^5.3.0",
-    "styled-components": "^4.2.1",
+    "styled-components": "^5.0.1",
```

Comment:
How sure are you this doesn't break something in govuk-react?

Reply:
![image](https://user-images.githubusercontent.com/7834222/83415151-8ec91500-a416-11ea-9b0d-f1ebf0fc55b6.png)
That's from the horse's mouth, but also I have tested the changes.

Reply:
Hmm okay. Weird versioning...